### ISSUE TITLE
does learning rate affect much for further finetune?

### ISSUE THREAD

**Issue by zeyofu** [DAMO-NLP-SG/VideoLLaMA3#74]
I'm finetuning beyond stage4 and using the learning rates listed in paper for stage4. I wonder if you have any intuition how I should tune these learning rates (because I have limited computation resource), and whether this would change the performance a lot on video understanding tasks.
